### DISCUSSION TITLE
Sleap 1.4.1?

### DISCUSSION THREAD

**Discussion by astoicame** [talmolab/sleap#1864] [General]
Hi Talmo Sleap team! I am currently using sleap 1.3.1 and am going to update to 1.3.3, but I am interested in your upcoming release. What is the timeline like for the official release of 1.4.1? Curious to know if I should delay my analysis if the release will be soon.

**Comment by talmo** [comment]
We are actively working on it and hope to have it out in the coming week(s)!

My recommendation would be to update to 1.3.3 for now, but you're welcome to try out the [1.4.1 pre-release](https://github.com/talmolab/sleap/releases/tag/v1.4.1a2).

@eberrigan @roomrys
